src/status_im/contexts/wallet/events.cljs

```diff
@@ -416,13 +416,13 @@
        (log/info "[wallet] Test network enabled: " (boolean test-networks-enabled?))
        (log/info "[wallet] Goerli network enabled: " (boolean is-goerli-enabled?)))
      {:db (assoc-in db [:wallet :statuses :blockchains] chains)
-      :fx (when chains-down?
-            [[:dispatch
+      :fx [(when chains-down?
+             [:dispatch
               [:toasts/upsert
                {:id       :chains-down
                 :type     :negative
                 :text     (i18n/label :t/provider-is-down {:chains chain-names})
-                :duration 10000}]]])})))
+                :duration 10000}]])]})))
```

Comment:
maybe change to 1/2  secs while you make this change.
cc @BalogunofAfrica

Reply:
@J-Son89, just to confirm, which value do you want here? Currently it's 10 seconds. Do you mean it should be 5 seconds?

Reply:
Just anything shorter really, 5 is fine

Reply:
Done